_posts/2025-04-28-sparse-autodiff.md

```diff
@@ -488,8 +489,59 @@ Instead, it returns an empty set.
 *TODO: switch to multivariate function, quickly discuss resulting Jacobian.*
 <!-- TODO -->
 
+<!-- Coloring techniques? -->
 ### Matrix coloring
 
+Coloring techniques, such as row and column colorings, are used to determine the seed vectors needed to compute the minimal number of directional derivatives required to recover all the non-zero entries of Jacobians.
+A seed vector is a linear combination of the rows or columns of the identity matrix.
+When a directional derivative of a vector-valued function is taken along this seed vector, the result is a linear combination of the corresponding rows or columns of the Jacobian.
+
+Theses methods relies on reverse or forward automatic differentiation (AD) to efficiently recover the compressed rows or columns of the Jacobian.
+This is particularly valuable when dealing with sparse Jacobians, as it enables significant compression without overlap among the non-zero entries.
```

Comment:
Maybe this concept of a seed vector could be moved into the "Leveraging structure" section?

I feel like there is some overlap here.